docs/main/user-guides/replicated-storage-user-guide/advanced-operations/monitoring.md

```diff
@@ -101,4 +101,109 @@ Upon successful integration of the exporter with the Prometheus stack, the metri
 | kubelet_volume_stats_inodes_used | Gauge | Integer | The total number of inodes that have been utilized to store metadata. |
 
 
+## Performance Monitoring Stack
+
+Earlier, the pool capacity/state stats were exported, and the exporter used to cache the metrics and return when Prometheus client queried. This was not ensuring the latest data retuns during the Prometheus poll cycle.
+
+This feature exports performace stats for Volume, Replica and DiskPool resource. Mayastor Prometheus exporter runs along with IO engine container and exposes Prometheus counters. In addition to that, exporter does not pre-populate the metrics. It queries the IO engine, updates cache, and returns the latest metrics when the Prometheus client queries the exporter.
```

Comment:
performance*

Remove the "Mayastor Prometheus exporter..." as we use the same exporter which previously existed and exported other metrics.

We need a bit of re-formatting here.

In addition to the capacity and state metrics, the metrics exporter also exports performance statistics for pools, volumes and replicas as Prometheus counters. The exporter does not pre-fetch or cache the metrics, it polls the IO engine inline with the Prometheus client pooling cycle.